network/p2p/libp2pNode.go

```diff
@@ -274,8 +274,8 @@ func (n *Node) tryCreateNewStream(ctx context.Context, identity flow.Identity, m
 
 		// if this is a retry attempt, wait for some time before retrying
 		if retries > 0 {
-			// choose a random interval between 0 and 5 ms to retry
-			r := rand.Intn(5)
+			// choose a random interval between 10 and 20 ms to retry
```

Comment:
just an fyi - Raul had asked me to add this randomness when retrying because it could be the case that two nodes retry at the same time and that would result in a wierd libp2p corner case.